atomsci/ddm/utils/datastore_functions.py

```diff
@@ -2059,12 +2066,12 @@ def get_key_val(metadata, key=None):
         returns When key is provide, returns value for matching key if found, None otherwise
         returns dictionary for the list of key,value pairs when a query key is not provided.
     """
-    if key == None :
+    if key is None :
```

Comment:
test?

Reply:
This will need to wait for a later release when the datastore is back up.